speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/raceresultsmenus.cpp

```diff
@@ -528,7 +528,7 @@ rmQualifResults(void *prevHdle, tRmInfo *info, const char* pszTitle, int start)
     GfuiMenuCreateButtonControl(rmScrHdle, hmenu, "ContinueButton", prevHdle, GfuiScreenReplace);
     
     // SIMULATED DRIVING ASSISTANCE: Add "Quit" button
-    GfuiMenuCreateButtonControl(rmScrHdle, hmenu, "QuitButton", EndExperimentInit(), GfuiScreenActivate);
+    GfuiMenuCreateButtonControl(rmScrHdle, hmenu, "QuitButton", EndExperimentInit(RACE_EXIT), GfuiScreenActivate);
```

Comment:
I'm pretty sure this code never gets called anymore since we dont have qualifiers in our quickrace.

Reply:
I did not write this part of the code, I only changed so that it worked, it will probably be deleted in another branch (change menu layout)